tensorflow/core/kernels/data/sampling_dataset_op.cc

```diff
@@ -0,0 +1,202 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
this kernel should go into `tensorflow/core/kernels/data/experimental`

Reply:
kernel had been moved to experimental